test/gurka/test_matrix.cc

```diff
@@ -41,7 +41,7 @@ void check_matrix(const rapidjson::Document& result,
                         std::to_string(i % origin_td.Size());
       EXPECT_NEAR(v.GetObject()["distance"].GetFloat(), exp_dists[i], 0.01) << msg;
       if (valid_traffic) {
-        EXPECT_TRUE(v.GetObject().HasMember("date_time")) << msg;
+        ASSERT_TRUE(v.GetObject().HasMember("date_time")) << msg;
```

Comment:
some small improvements, the tests were failing locally, so I made them a bit more comprehensible.